test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.AttributesWithBooleanValues.html

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
If the CI is broken, fix it immediately. Otherwise this seems to be "fallout" from the unnecessary `Trim()` expressions.